demucs/spec.py

```diff
@@ -7,7 +7,7 @@
 """Conveniance wrapper to perform STFT and iSTFT"""
 
 import torch as th
-
+from .stft import demucs_stft
 
 def spectro(x, n_fft=512, hop_length=None, pad=0, onnx_exportable=False):
```

Comment:
suuuuper nit: PEP recommends 2 lines after import sections. This change would usually fail linters. I might try and add a basic one with pre-commit at some point! 
```suggestion
from .stft import demucs_stft


def spectro(x, n_fft=512, hop_length=None, pad=0, onnx_exportable=False):
```